spec/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
+  # (RE)XML related helper methods.
+  module XML
+    # Converts `REXML::Attributes` into a `Hash`.
+    #
+    # @param [REXML::Attributes] attributes The attributes to convert
+    #   into a Hash.
+    # @return [Hash] the attribute keys mapped to the attribute values
+    def attributes_to_hash(attributes)
+      attributes.to_enum.to_h
```

Comment:
Awesome, Travis spotted this failure on 1.9.3 and 2.0 where `Enumerator#to_h` was not yet defined. I'll make this more generic later today.